lua/nvim-treesitter/parsers.lua

```diff
@@ -1310,6 +1310,14 @@ list.rego = {
   maintainers = { "@FallenAngel97" },
 }
 
+list.requirements = {
+  install_info = {
+    url = "https://github.com/ObserverOfTime/tree-sitter-requirements",
+    files = { "src/parser.c" },
+  },
+  maintainers = { "@ObserverOfTime" },
```

Comment:
Maybe add a `readme_note` explaining what this is for? ("requirements" is _very_ generic...) Does this need to register a filetype?

Reply:
> Does this need to register a filetype?

I've submitted a PR to Vim.

Reply:
I wouldn't mind leaving this open until that's merged.

Reply:
It's ready now.